src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -1279,7 +1280,42 @@ private void Abort(Exception abortException)
             return LifetimeExpired(nowTicks, connectionLifetime);
         }
 
-        private void AbortStreams(int lastValidStream, Exception abortException)
+        private void AbortStreams(Exception abortException)
+        {
+            lock (SyncObject)
+            {
+                if (NetEventSource.IsEnabled) Trace($"{nameof(abortException)}={abortException}, {nameof(_lastValidStreamId)}={_lastValidStreamId}");
```

Comment:
Is it worthwhile to actually log _lastValidStreamId here now?

Reply:
mixed feelings, i suspect it might be useful in case there are any races between Abort and GOAWAY but don't actually have specific usage. Will remove and we can add it back if useful in the future